html/host.html

```diff
@@ -6,63 +6,105 @@
+                <option>[4] Investments</option>
+                <option>[4] Predator Prey Game</option>
+                <option>[4] Root Beer Game</option>
+                <option>[3] Bug Hunters Camouflage</option>
+                <option>[3] Disease With Android Avoidance</option>
+                <option>[3] Minority Game</option>
+                <option>[3] Tragedy of the Commons</option>
+                <option>[3] Walking</option>
+                <option>[2] Disease Doctors</option>
+                <option>[2] Memory</option>
+                <option>[2] Oil Cartel Alternate</option>
+                <option>[2] Public Good</option>
+                <option>[1] BeeSmart</option>
+                <option>[1] Bug Hunters Adaptations</option>
+                <option>[1] Bug Hunters Competition</option>
+                <option>[1] Dice Stalagmite</option>
+                <option>[1] Fish Spotters</option>
+                <option>[1] Gridlock</option>
+                <option>[1] Guppy Spots</option>
+                <option>[1] PANDA BEAR</option>
+                <option>[1] Polling</option>
+                <option>[1] Polling Advanced</option>
+                <option>[1] Prisoners Dilemma</option>
+
+                <option>[a] Oil Cartel</option> <!-- requires narrowcast plots -->
+
+                <option>[0] Restaurants</option> <!-- makes no distinction between AI restaurants and player-owned ones -->
+
+                <option>[0] Bird Breeders</option> <!-- requires spectator clients -->
+                <option>[0] Gridlock Alternate</option> <!-- requires spectator clients -->
+
+                <option>[0] Client Overrides Example</option> <!-- requires view overrides -->
+                <option>[0] Critter Designers</option> <!-- requires view overrides -->
+                <option>[0] Current in a Wire</option> <!-- requires view overrides -->
+                <option>[0] MTG 1 Equal Opportunities</option> <!-- requires view overrides -->
+                <option>[0] MTG 2 Random Assignment</option> <!-- requires view overrides -->
+                <option>[0] MTG 3 Feedback Loop</option> <!-- requires view overrides -->
+                <option>[0] Sampler</option> <!-- requires view overrides -->
+              </select>
+            </div>
+            <p id="activity-text-short" class="activity-text-short">
+              This is a placeholder for a condensed activity description.
+            </p>
+            <input type="button" id="read-more-button" class="read-more-button" value="Read More"></input>
+          </div>
+
+          <div class="image-wrapper">
+            <img
+              id="preview-image"
+              class="preview-image"
+              src=" http://ccl.northwestern.edu/netlogo/models/models/Curricular%20Models/epiDEM/epiDEM%20Basic.png"
+              alt="Activity Image"
+            >
+          </div>
+        </div>
+
+        <div class="form-input-wrapper">
```

Comment:
Please add a margin below this.  It's uncomfortable that these controls brush right up against the bottom of the page.

Reply:
Added a `20px` margin - hopefully it looks better-ish/okay on your machine now as well.